types/mojang-net/index.d.ts

```diff
@@ -0,0 +1,172 @@
+// Type definitions for Minecraft Bedrock Edition script APIs (experimental) 0.1
+// Project: https://docs.microsoft.com/minecraft/creator/
+// Definitions by: Jake Shirley <https://github.com/JakeShirley>
+//                 Mike Ammerlaan <https://github.com/mammerla>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* *****************************************************************************
+   Copyright (c) Microsoft Corporation.
+   ***************************************************************************** */
+/**
+ * The `mojang-net` module contains types for executing
+ * HTTP-based requests.
+ *
+ * Manifest Details
+ * ```json
+ * {
+ *   // mojang-net
+ *   "uuid": "777b1798-13a6-401c-9cba-0cf17e31a81b",
+ *   "version": [ 0, 1, 0 ]
+ * }
+ * ```
+ *
+ */
+import * as mojangminecraftserveradmin from 'mojang-minecraft-server-admin';
+export enum HttpRequestMethod {
+    /**
+     * Represents the method for an HTTP PUT request. GET requests
+     * are commonly used to retrieve information about a resource
+     * at the specified URI.
+     */
+    POST = 0,
+    /**
+     * Represents the method for an HTTP PUT request. PUT requests
+     * are commonly used to update a single resource that already
+     * exists in a resource collection.
+     */
+    PUT = 1,
+    /**
+     * Represents the method for an HTTP PUT request. POST requests
+     * are commonly used to create a new resource that is a
+     * subordinate of the specified URI.
+     */
+    GET = 2,
+    /**
+     * Represents the method for an HTTP HEAD request. HEAD
+     * requests are similar to a GET request, but are commonly used
+     * to retrieve just the HTTP response headers from the
+     * specified URI, and not the body contents.
+     */
+    DELETE = 3,
+    /**
+     * Represents the method for an HTTP PUT request. GET requests
+     * are commonly used to retrieve information about a resource
+     * at the specified URI.
+     */
+    HEAD = 4,
+}
+export class HttpClient {
+    /**
+     * @remarks
+     * Cancels all pending requests.
+     * @param reason
```

Comment:
Should this have a description?

Reply:
I think `@remarks` is non-JSDoc, so can be just removed and JSDoc always takes whatever given as description (so @description is not required)